canonical-web-and-design/maas-ui
# Fix KVM list QA issues

## Done
- Fixed small cores chart alignment.
- Added popover to RAM column.
- Fixed LXD pod total storage calculation.

## QA
- Check that the small cores chart aligns to the top of the RAM and storage meters.
- Hover over a RAM meter and check that a popover shows up with used/free/total info. If the pod has a memory overcommit ratio it should show next to the total.
- Check that the storage meter on a LXD pod accurately reflects the numbers.

## Fixes
Fixes #1109 

## Screenshots
### Small cores chart
![2020-05-12_11-57](https://user-images.githubusercontent.com/25733845/81630228-6322c280-9448-11ea-96e9-78765b0b9d8b.png)

### RAM popover
![2020-05-12_11-58_1](https://user-images.githubusercontent.com/25733845/81630234-66b64980-9448-11ea-9539-e26fb35b59d5.png)

### Storage meter
![2020-05-12_11-58](https://user-images.githubusercontent.com/25733845/81630236-6918a380-9448-11ea-8886-538073588d10.png)
